src/extension/src/ActionHandler.py

```diff
@@ -88,16 +92,21 @@ def setup_telemetry(self):
             self.logger.log_error(err_msg)
         else:
             self.logger.log("The minimum Azure Linux Agent version prerequisite for Linux patching was met.")
-            self.logger.telemetry_writer.events_folder_path = events_folder
+            self.telemetry_writer.events_folder_path = events_folder
+            # As this is a common function used by all handler actions, setting operation_id such that it will be the same timestamp for all handler actions, which can be used for identifying all events for an operation.
+            # NOTE: Enable handler action will set operation_id to activity_id from config settings. And the same will be used in Core.
+            self.telemetry_writer.set_operation_id(self.timestamp_for_telemetry)
```

Comment:
NOTE: Since this setup is also used for enable, operationid will be set to the common timestamp initially and later set to activity id within enable's action handler. This leads to the the initial setup operations for enable to have a timestamp while later operation will have activity_id. 
For eg: 
![image](https://user-images.githubusercontent.com/56841542/108562072-27106e00-72b4-11eb-9a5b-11e06501cbb9.png)


 Let me know if this is okay.